src/options/options.html

```diff
@@ -64,6 +64,15 @@ <h1 data-i18n="__MSG_titleGeneral__">Add-on</h1>
 							<span class="helper-text" data-i18n="__MSG_optionPopupIconColoredDescr__">Shows a colored icon instead of the black/white icon in the toolbar.</span>
 						</div>
 					</li>
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="contextMenuEnabled" name="contextMenuEnabled">
+							<label for="contextMenuEnabled">Enable QR code context menu item</label>							
+						</div>
+						<div class="line indent">
+							<span class="helper-text">Show context menu item to create QR code for the current page.</span>
```

Comment:
Ah misse done thing: let's also talk about how to best communicate this setting only affects _that one menu entry that you've added_. Otherwise people could get confused that menu entries are still shown when this is not selected, which is, however, correct? Confusion is bad for UX though. So maybe like this?

```suggestion
							<label for="contextMenuEnabled">Show general QR code generation context menu item</label>							
						</div>
						<div class="line indent">
							<span class="helper-text">Shows a context menu item to create QR code for the current page, in addition to links and text selections.</span>
```